src/lib/exposes.ts

```diff
@@ -634,7 +634,7 @@ export const presets = {
     switch_: () => new Switch(),
     // Specific
     ac_frequency: () => new Numeric('ac_frequency', access.STATE).withLabel('AC frequency').withUnit('Hz').withDescription('Measured electrical AC frequency'),
-    action: (values: string[]) => new Enum('action', access.STATE, values).withDescription('Triggered action (e.g. a button click)'),
+    action: (values: string[]) => new Enum('action', access.STATE, values).withDescription('Triggered action (e.g. a button click)').withCategory('diagnostic'),
```

Comment:
Why is this a `diagnostic` `category`?

Reply:
Because action isn’t something to show on a dashboard, as it will be cleared instantly. The main function of the entity_category is exactly that. To sort by get/set and to hide almost everything unnecessary from the dashboards by default